/**:
  ros__parameters:
    save_dir: ""
    use_phoxi: true
    use_external_rgb: true
    use_external_depth: true
